pymobiledevice3/__main__.py

```diff
@@ -90,7 +90,8 @@ def cli():
         traceback.print_exc()
     except LaunchApplicationTimeoutError:
         logger.error('Unable to launch application (try to unlock device)')
-
+    except WebInspectorNotEnabledError:
```

Comment:
The main try except handles more low level exception, you should it put it under webinspector

Reply:
changed to webinspector decorator